sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -581,8 +585,11 @@ export class EventHubSender extends LinkEntity {
       operation: sendEventPromise,
       connectionId: this._context.connectionId,
       operationType: RetryOperationType.sendMessage,
-      times: Constants.defaultRetryAttempts,
-      delayInSeconds: Constants.defaultDelayBetweenOperationRetriesInSeconds + jitterInSeconds
+      times: options && options.retryAttempts ? options.retryAttempts : Constants.defaultRetryAttempts,
```

Comment:
`options && options.retryAttempts` is "falsy", if the provided value was 0.
Use the user provided value only if it is greater than 0.
Check with @bterlson if we want `retryAttempts` as exposed to the user to mean the total number of attempts or the number of attempts after the first failure